tests/4.5/target_update/test_target_update_from.c

```diff
@@ -58,16 +56,9 @@ int main() {
 }// end target-data
```

Comment:
Indentation width is too long on the lines above here

Reply:
Also after linee 48, we should check for the value of b is correct, on the host side.